aries_cloudagent/did/tdw/routes.py

```diff
@@ -0,0 +1,84 @@
+"""DID TDW routes."""
+
+from http import HTTPStatus
+
+from aiohttp import web
+from aiohttp_apispec import docs, request_schema, response_schema
+from marshmallow import fields
+
+from ...admin.decorators.auth import tenant_authentication
+from ...admin.request_context import AdminRequestContext
+from ...did.tdw.tdw_manager import DidTdwManager
+from ...messaging.models.openapi import OpenAPISchema
+
+
+class DidTdwResolveSchema(OpenAPISchema):
+    """Parameters and validators for create DID endpoint."""
+
+    did = fields.Str(
+        required=True,
+        metadata={
+            "example": "did:tdw:QmRHpkso6BM1jRvGee5VoiS3a6wUfPAfuPnyGRBFo4rL6k:example.com",  # noqa: E501
+            "description": "Did TDW to resolve",
+        },
+    )
+
+
+class DidTdwResolveResponse(OpenAPISchema):
+    """Response schema for create DID endpoint."""
+
+    did_doc = fields.Dict(
```

Comment:
I'm thinking about just returning the diddoc dict here instead of inside this key.